content/en/docs/concepts/services-networking/_index.md

```diff
@@ -7,44 +7,195 @@ description: >
+{{< /note >}}
+
+## Requirements
+
+Every [Pod](/docs/concepts/workloads/pods/) in your cluster gets its own unique cluster-wide IP address called a _pod IP_. 
+
+If you have deployed an [IPv4/IPv6 dual stack](/docs/concepts/services-networking/dual-stack/) cluster,
+then you - or your network plugin(s) - must allocate pod IPs for IPv4 and IPv6 for each pod.
+
+This is performed per [_address family_](https://www.iana.org/assignments/address-family-numbers/address-family-numbers.xhtml), with one for IPv4 addresses and one for IPv6 addresses. You can assign an Ethernet MAC address to each Pod. Kubernetes does not require that a Pod has a unique identity at the data-link layer.
+
+Kubernetes imposes the following requirements on any networking
 implementation (barring any intentional network segmentation policies):
 
-   * pods can communicate with all other pods on any other [node](/docs/concepts/architecture/nodes/) 
-     without NAT
-   * agents on a node (e.g. system daemons, kubelet) can communicate with all
-     pods on that node
-
-Note: For those platforms that support `Pods` running in the host network (e.g.
-Linux), when pods are attached to the host network of a node they can still communicate 
-with all pods on all nodes without NAT.
-
-This model is not only less complex overall, but it is principally compatible
-with the desire for Kubernetes to enable low-friction porting of apps from VMs
-to containers.  If your job previously ran in a VM, your VM had an IP and could
-talk to other VMs in your project.  This is the same basic model.
-
-Kubernetes IP addresses exist at the `Pod` scope - containers within a `Pod`
-share their network namespaces - including their IP address and MAC address.
-This means that containers within a `Pod` can all reach each other's ports on
-`localhost`. This also means that containers within a `Pod` must coordinate port
-usage, but this is no different from processes in a VM.  This is called the
-"IP-per-pod" model.
-
-How this is implemented is a detail of the particular container runtime in use.
-
-It is possible to request ports on the `Node` itself which forward to your `Pod`
-(called host ports), but this is a very niche operation. How that forwarding is
-implemented is also a detail of the container runtime. The `Pod` itself is
-blind to the existence or non-existence of host ports.
+* Containers in same Pod can communicate with each other.
+* Pods can communicate with all other Pods on the same or separate [nodes](/docs/concepts/architecture/nodes/)
+without network address translation (NAT).
+* Pods can communicate with other Pods on the same or separate nodes using layer 2 bridging, native layer 3 (IPv4, IPv6) networking, MPLS, segment routing, tunneling, or other techniques for conveying packets from one Pod to another.
```

Comment:
I prefer the existing paragraph, why do we need to enumerate all the network technologies?

Reply:
Some new readers from the networking world or operators who have deployed SRv6, for example, may wish to know if a protocol can accommodate K8s network model. Vendors, operators, students, network architects, etc. have discussed these topics.

Reply:
Also Kubernetes added IPv6 after IPv4. See your point though.